Makefile

```diff
@@ -32,11 +32,15 @@ image_base="ubuntu"
 image_tag="20.04"
 endif
 
+buildx=0
+cache_from=type=local,src=/tmp/.buildx-cache
+cache_to=type=local,dest=/tmp/.buildx-cache
 ### function for building
 ### $(1) is name
 ### $(2) is dockerfile filename
 ### $(3) is package type
 ### $(4) is code path
+ifneq ($(buildx), "true")
```

Comment:
@qsliu2017 The https://github.com/api7/apisix-build-tools/pull/74/checks?check_run_id=3520664903#step:6:6 showed that the `buildx=true` did work as expected. So I guess you should change this to `ifneq ($(buildx), true)`. Thanks.

Reply:
@qsliu2017 Hi, in plan we are going to prepare for a new release. Since your PR will significantly reduce the CI time, would you mind make the above changes ASAP? Much appreciated.